### ISSUE TITLE
Wrong HACS icon sizes in Firefox

### ISSUE THREAD

**Issue by Eyco1981** [home-assistant/frontend#10556]
### Checklist

- [X] I have updated to the latest available Home Assistant version.
- [X] I have cleared the cache of my browser.
- [X] I have tried a different browser to see if it is related to my browser.

### Describe the issue you are experiencing

When using Firefox the HACS icon is scaling bigger in the sidebar and e.g. as icon in entity and element cards when using Firefox. In Chrome and the HA mobile app the icons look as expected. Wiping of browser cache and deleting HA related cookies did not change that behavior.

### Describe the behavior you expected

Same icon scaling as other ones.

### Steps to reproduce the issue

1. Load HA web interface via IP address, homeassistant.local or the Nabu Casa URL (using Firefox)
2.  Login
3. (additional) Use sensor.hacs in entity or element cards
...

### What version of Home Assistant Core has the issue?

core-2021.11.1

### What was the last working version of Home Assistant Core?

core-2011.10.6

### In which browser are you experiencing the issue with?

Firefox 94.0.1 (64-Bit) and Firefox Mobile 93.2.0 (Android)

### Which operating system are you using to run this browser?

Windows 11 (Version 21H2 / Build 22000.282) and Android 0

**Comment by ludeeus** [comment]
The title and description is misleading, it's not limited to HACS 
![image](https://user-images.githubusercontent.com/15093472/140611755-714f22d1-f2d4-47cd-8c4e-d39ee39a7710.png)
(image taken from the forum, personally I can't replicate it)

**Comment by Eyco1981** [comment]
Have updated the title. It just happend to me with the HACS icon. But after some trying, it seems other icons than the HA and MDI ones seem to be effected (e.g. the fontawesome icons).

**Comment by hanscats** [comment]
This happens to me when installing Cupertino Icons as a integration. When I disable the integration the icons are normal size again. Do you by chance also have Cupertino Icons as a integration, @Eyco1981 ?

**Comment by Stooovie** [comment]
The image is mine, Fontawesome and phu: icons (I can't find what those are atm, but it's either from HACS or an official integration) are bigger than the default mdi: icons. It's a new issue in HA 2021.X.

**Comment by ruifung** [comment]
For me, a workaround seems to be disabling the fontawesome icons integration.

**Comment by Stooovie** [comment]
BTW the issue is only present on iOS, so the same webkit in all browsers as well as the HA app.

**Comment by Jbdawg** [comment]
@ruifung I disabled fontawesome and now icons are back to normal for me as well, thanks. Do we think this is a bug with fontawsome itself or the latest HA core release? I am not to bright when it comes to how integrations interact with HA.

**Comment by Stooovie** [comment]
No, it happens with any icon pack other than the default mdi icons. Not just Font Awesome.

**Comment by Stooovie** [comment]
> For me, a workaround seems to be disabling the fontawesome icons integration.

happens with all icons other than the default ones, such as BHA Custom icons and Custom brand icons from HACS. Not just Font Awesome icons.

**Comment by ruifung** [comment]
> 
> 
> > For me, a workaround seems to be disabling the fontawesome icons integration.
> 
> happens with all icons other than the default ones, such as BHA Custom icons and Custom brand icons from HACS. Not just Font Awesome icons.

Huh. So it seems like just having any icon pack installed causes it?

**Comment by Stooovie** [comment]
Yes. On my setup anyway. Uninstalling Font awesome, restarting and clearing cache didn't fix the issue.

**Comment by thomasloven** [comment]
This should be fixed in Fontawesome 2.1.5
https://github.com/thomasloven/hass-fontawesome/releases/tag/2.1.5

**Comment by Stooovie** [comment]
Confirm fixed. Thanks!
